css/style.css

```diff
@@ -20,9 +20,9 @@ header {
 	border: 2px solid;
 	background-color: #ef9166;
 	color: #F1F1F2;
-    border-radius: 25px;
-    width: 120px;
-    height: 60px;
+	border-radius: 25px;
+	width: 120px;
+	height: 60px;
```

Comment:
You'll see a bunch of these. I just fixed the indentation in your CSS file (some props were indented twice).